plugins/doc_fragments/ldap.py

```diff
@@ -12,12 +12,12 @@
 
 class ModuleDocFragment(object):
     # Standard LDAP documentation fragment
-    DOCUMENTATION = r'''
+    DOCUMENTATION = r"""
 options:
   bind_dn:
     description:
-      - A DN to bind with. If this is omitted, we'll try a SASL bind with the EXTERNAL mechanism as default.
-      - If this is blank, we'll use an anonymous bind.
+      - A DN to bind with. If this is omitted, it tries a SASL bind with the EXTERNAL mechanism as default.
+      - If this is blank, it uses an anonymous bind.
```

Comment:
If we wouldn't have https://docs.ansible.com/ansible/devel/dev_guide/style_guide/index.html#stylistic-cheat-sheet I'd say passive voice would be better here... The 'it' sounds strange.

Hmm, thinking more about it, how about:
```suggestion
      - A DN to bind with. Try to use a SASL bind with the EXTERNAL mechanism as default when this parameter is omitted.
      - Use an anonymous bind if the parameter is blank.
```
Then it's active and doesn't use 'it' for the module/plugin/....